docs/source/yelpsoa_configs.rst

```diff
@@ -157,6 +157,169 @@ Marathon, Chronos, Tron, or ``paasta remote-run``.
+instance MAY have:
+
+  * Anything in the `Common Settings`_.
+
+  * Only ``pool`` from `Placement Options (Constraints)`_.
+
+  * ``cap_add``: List of capabilities that are passed to Docker. Defaults
+    to empty list. Example::
+
+      "cap_add": ["IPC_LOCK", "SYS_PTRACE"]
+
+  * ``instances``: Kubernetes will attempt to run this many instances of the Service
+
+  * ``min_instances``: When autoscaling, the minimum number of instances that
+    marathon will create for a service. Defaults to 1.
+
+  * ``max_instances``: When autoscaling, the maximum number of instances that
+    marathon will create for a service
+
+  * ``registrations``: A list of SmartStack registrations (service.namespace)
+    where instances of this PaaSTA service ought register in. In SmartStack,
+    each service has difference pools of backend servers that are listening on
+    a particular port. In PaaSTA we call these "Registrations". By default, the
+    Registration assigned to a particular instance in PaaSTA has the *same name*,
+    so a service ``foo`` with a ``main`` instance will correspond to the
+    ``foo.main`` Registration. This would correspond to the SmartStack
+    namespace defined in the Registration service's ``smartstack.yaml``. This
+    ``registrations`` option allows users to make PaaSTA instances appear
+    under an *alternative* namespace (or even service). For example
+    ``canary`` instances can have ``registrations: ['foo.main']`` to route
+    their traffic to the same pool as the other ``main`` instances.
+
+  * ``container_port``: Specify the port to expose when in ``bridge`` mode.
+    Defaults to ``8888``.
+
+  * ``bounce_method``: Controls the bounce method; see `bounce_lib <generated/paasta_tools.bounce_lib.html>`_
+
+  * ``bounce_health_params``: A dictionary of parameters for get_happy_tasks.
+
+    * ``check_haproxy``: Boolean indicating if PaaSTA should check the local
+      haproxy to make sure this task has been registered and discovered
+      (Defaults to ``True`` if service is in SmartStack)
+
+    * ``min_task_uptime``: Minimum number of seconds that a task must be
+      running before we consider it healthy (Disabled by default)
+
+    * ``haproxy_min_fraction_up``: if ``check_haproxy`` is True, we check haproxy on up to 20 boxes to see whether a task is available.
+      This fraction of boxes must agree that the task is up for the bounce to treat a task as healthy.
+      Defaults to 1.0 -- haproxy on all queried boxes must agree that the task is up.
+
+  * ``bounce_margin_factor``: proportionally increase the number of old instances
+    to be drained when the crossover bounce method is used.
+    0 < bounce_margin_factor <= 1. Defaults to 1 (no influence).
+    This allows bounces to proceed in the face of a percentage of failures.
+    It doesn’t affect any other bounce method but crossover.
+    See `the bounce docs <bouncing.html>`_ for a more detailed description.
```

Comment:
I think this needs update as well. I deprecated upthedown but we need to update/rephrase here and there. For example, in "The bounce_margin_factor setting in marathon yaml files .",   we can use marathon/kubernetes instead. Also, we might want to remove chronos section? Leaving it here will make me feel that this doc is not trustworthy.